docs/src/getstarted/hello-world.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Hello World Quickstart Guide"
+description: "This quickstart guide will demonstrate how to setup, build, and deploy your first Solana program in your browser with Solana Playground."
+keywords: "playground, solana pg, on chain, rust, native program, tutorial, intro to solana development, blockchain developer, blockchain tutorial, web3 developer"
+---
+
+For this quickstart guide, we will use [Solana Playground](https://beta.solpg.io), a browser the based IDE, to develop and deploy our Solana program. To use it, you do **NOT** have to install any software on your computer. Simply open Solana Playground in your browser of choice, and you are read to write and deploy Solana programs.
+
+## What you will learn
+
+- How to get started with Solana Playground
+- How to create a Solana wallet on Playground
+- How to program a basic Solana program in Rust
+- How to build and deploy a Solana Rust program
+
+## Import our example project
+
+In a new tab in your browser, open our example "_Hello World_" project on Solana Playground: https://beta.solpg.io/6314a69688a7fca897ad7d1d
+
+Next, import the project into your local workspace by clicking the "**Import**" icon and naming your project `hello_world`.
+
+![](/img/quickstarts/solana-get-started-import-on-playground.png)
+
+## Create a Playground wallet
+
+Normally with [local development](./local.md), you will need to create a file system wallet for use with the Solana CLI. But with the Solana Playground, you only need to click a few buttons to create a browser based wallet.
+
+> **Note:**
+> Your _Playground Wallet_ will be saved in your browser's local storage. Clearing your browser cache will remove your saved wallet. When creating a new wallet, you will have the option to save a local copy of your wallet's keypair file.
+
+Click on the red status indicator button at the bottom left of the screen, (optionally) save your wallet's keypair file to your computer for backup, then click "**Continue**".
+
+After your Playground Wallet is created, you will notice the bottom of the window now states your wallet's address, your SOL balance, and the Solana cluster you are connected to (usually Devnet by default, but Testnet is also acceptable).
```

Comment:
Testnet is meant to be for performance testing. Let's remove the "Testnet is also acceptable" and actually recommend devnet or local if the dev has it set up.